### PR TITLE
Added descriptions to projects

### PR DESCRIPTION
**Pull request by MargoWM**

### CODE REVIEW

**Comment by MargoWM** [comment]
In Alinka application I added two links for partners' websites. I'm not sure if it is properly?
I didn't add images for projects. We need to choose them.
I don't know how to check versions of used technologies

File: frontend/src/assets/projects.js [hunk 1]
```diff
@@ -26,61 +26,61 @@ const projects = [
...
         version: '',
         documentation: ''
       }
     ],
     website: ''
   },
   {
-    description: '',
-    githubLink: '',
+    description: ' Aplikacja Alinka powstała w celu optymalizacji codziennej pracy w poradniach psychologiczno-pedagogicznych. Jako byli pracownicy takiej poradni doskonale rozumiemy jakiego nakładu pracy wymaga wydanie orzeczenia czy sporządzenie protokołu z posiedzenia komisji. Aplikacja Alinka jest odpowiedzią na trudności z jakimi spotykają się w codziennej pracy pracownicy administracji poradni. Nazwa aplikacji: „Alinka” to imię prawdziwej osoby, która musi wykonywać tę nudną, ale bardzo ważną pracę. Mamy nadzieję, że któregoś dnia ta aplikacja pomoże w jej pracy. :)',
```
Comment (w1stler) [inline frontend/src/assets/projects.js:54]: Redundant space in the beginnig.

Comment (w1stler) [reply frontend/src/assets/projects.js:54]: (...) jest odpowiedzią na **trudności, z** jakimi spotykają się (...)

add comma

File: frontend/src/assets/projects.js [hunk 2]
```diff
@@ -98,39 +98,39 @@ const projects = [
         documentation: ''
       }
     ],
-    website: ''
+    website: 'http://bankempatii.pl/'
   },
   {
-    description: '',
-    githubLink: '',
+    description: 'Oryginalny StreetMix jest dziełem CodeforAmerica, które w 2013 roku podczas hacknightu postanowiło stworzyć narzędzie, które pomoże miejskim planistom w przedstawianiu koncepcji ich planu na rewitalizacje miejskiej przestrzeni. Najczęstszym rozwiązaniem po jakie sięgają planiści jest wykonanie makiet z papieru, które zwizualizują krajobraz.  Tworząc internetową wersję tego działania, planiści mogą dotrzeć do szerszego grona odbiorców niż podczas samych spotkań, a także pozwolić członkom społeczności na dzielenie się i remiksowanie swoich dzieł. Jest to jeden z pierwszych naszych projektów, które mieliśmy przyjemność przetłumaczyć tak, aby można było wykorzystać te rozwiązani na polskim gruncie.',
```
Comment (w1stler) [inline frontend/src/assets/projects.js:104]: - Streetmix, not StreetMix
- Code for America - I think that's the correct name (with spaces).
- "rewlitalizacje" or "rewitalizację"
- double space before: "Tworząc internetową (...)"
- "rozwiązani" - missing "e"

**Comment by MargoWM** [comment]
@w1stler and what about the technologies we use. Are they correct?

**Pull request closed without merging**